bin/add-cmderplugin.ps1

```diff
@@ -0,0 +1,161 @@
+<#
```

Comment:
Ok, the readme doesn't say anything else than "it will be in path" and it already has the alias.bat in it... Still seems untidy (like the stuff in config until it got mostly reorganized), but ok... :-)